src/store/auth.js

```diff
@@ -1,59 +1,67 @@
-import vueAuthInstance from '../services/auth.js'
+import {
+  initVueAuthenticate
+} from '../services/auth.js'
+
+let vueAuthInstance
 
 export default {
   state: {
-    user: {
-      displayname: '',
-      email: 'x'
-    },
-    isAuthenticated: vueAuthInstance.isAuthenticated()
+    displayname: '',
+    email: 'x',
+    isAuthenticated: false
+  },
+  getters: {
+    isAuthenticated: state => {
+      return state.isAuthenticated
+    }
   },
-
   mutations: {
-    isAuthenticated (state, payload) {
+    IS_AUTHENTICATED(state, payload) {
```

Comment:
Is this some vuex naming convention? Uppercase functions look :ghost:

Reply:
internal usage:
https://github.com/serv3r/phoenix/blob/master/apps/files/src/store.js
external recommendation:
https://vuejs.org/v2/style-guide/#Non-flux-state-management-use-with-caution